### PR TITLE
Release/1.2.34

### PR DESCRIPTION
**Pull request by sacOO7**
Added support for no connection serial

### CODE REVIEW

**Review by ttypic — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+
+[Full Changelog](https://github.com/ably/ably-java/compare/v1.2.33...v1.2.34)
+
+**Fixed bugs:**
+
+- Should send `DETACH` after receiving `ATTACHED` while in the `DETACHING` or `DETACHED` state \(`RTL5k`\) [\#846](https://github.com/ably/ably-java/issues/846)
+
+**Closed issues:**
+
+- LocalDevice reset will cause ClassCastException [\#985](https://github.com/ably/ably-java/issues/985)
```
Comment (maratal) [inline CHANGELOG.md:13]: In ably-cocoa we remove all the noise here and only leave issues that make sense for the end-user if they choose to read this change log.

Comment (sacOO7) [reply CHANGELOG.md:13]: Done

**Review by maratal — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (maratal) [reply CHANGELOG.md:13]: Somehow GH doesn't show me `Resolve` button. Try to resolve it yourself @sacOO7

**Pull request merged**